helm/alfresco-content-services/templates/_helpers-search.tpl

```diff
@@ -41,9 +41,10 @@ Usage: include "alfresco-content-services.audit.elasticsearchUrl" $
   {{- else }}
     {{- with (index .Values "elasticsearch") }}
       {{- if .enabled }}
-        {{- $auditEsHost := printf "%s-%s" $.Release.Name (($.Values.global.elasticsearch).service.name | default "elasticsearch") }}
-        {{- $auditEsPort := ($.Values.global.elasticsearch).service.ports.restApi | default 9200 }}
-        {{- $auditEsProto := .protocol | default "http" }}
+        {{- $elasticCtx := dict "Values" . "Chart" $.Chart "Release" $.Release }}
+        {{- $auditEsHost := include "elastic.es-servicename" $elasticCtx }}
+        {{- $auditEsPort := 9200 }}
+        {{- $auditEsProto := "http" }}
```

Comment:
explain this, are we deprecating global values?

(remember to document all the breaking changes in the docs)